kindle_notes_to_md.py

```diff
@@ -4,47 +4,45 @@
 to a Markdown document
 '''
 import argparse
-from collections import OrderedDict
+import os
 import sys
 import traceback
+from collections import OrderedDict
 
+import pyperclip
 from bs4 import BeautifulSoup
-
 from eglogging import *
+
 logging_load_human_config()
 
 
 class Note:
   # highlight, possibly including a note
   def __init__(self):
-    self.text     = ''   # the book text that was highligthed
-    self.note     = ''   # any note I added
-    self.source   = ''   # info about the source of this note (location etc.)
-    self.location = None # int Location as given by Kindle
-
+    self.text = ''  # the book text that was highlighted
+    self.note = ''  # any note I added
+    self.source = ''  # info about the source of this note (location etc.)
+    self.location = None  # int Location as given by Kindle
```

Comment:
If we think about such things right now, I propose to use `attrs` library right away to make the syntax even more readable and elegant. Otherwise both code styles are ok for me.

Reply:
`attrs` is cool, if you'd like to keep working on this and create a PR that uses it that'd be great :)

Reply:
For this kind of alignment, I use editor extensions. Here's the one I'm using now in VS Code: https://marketplace.visualstudio.com/items?itemName=annsk.alignment